.github/workflows/golangci-lint.yml

```diff
@@ -33,15 +33,15 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with:
           # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
-          version: v1.45.2
+          version: v1.46.2
           
           skip-go-installation: true
 
           # Optional: working directory, useful for monorepos
           # working-directory: somedir
 
           # Optional: golangci-lint command line arguments.
-          args: --timeout=5m0s --tests=false
+          args: --timeout=5m0s
```

Comment:
Was linting tests problematic in the past? Some of them are no-brainers, like goimports, but maybe some other ones go overboard? Still, I would like to see if we can configure them individually if that is the case...